### ISSUE TITLE
QWIK ERROR Error: server can not rerender

### ISSUE THREAD

**Issue by GregOnNet** [BuilderIO/qwik#753]
### Qwik Version

0.0.34

### Operating System (or Browser)

Windows Chrome

### Node Version (if applicable)

16

### Which component is affected?

Qwik Runtime

### Expected Behaviour

Either the server does not an error or an error that helps to resolve the underlying problem.

### Actual Behaviour

Server throws an error but the App is running fine.

## Motivation

Currently, I do not understand the Console output of Qwik.
To me, it is hard to analyse what I have messed up.

Especially the messages `QWIK ERROR Error: server can not rerender` & `QWIK Dropping render. State changed during render` lacking some context, what I as a developer should do about it.

Thanks in advance for your help!

## Reproduce

```
git clone https://github.com/GregOnNet/kanban-board
cd kanban-board
npm i

# Console 1
npm run api: start

# Console 2
npm run app:start

# Open Browser at localhost:3000

# Refer to Consoe 2 displaying an error
```

## Error seen

```
QWIK ERROR Error: server can not rerender
    at logError (/home/gregonnet/workbench/projects/qwik-acadamy/kanban-board/node_modules/@builder.io/qwik/server.cjs:175:52)
    at Object.raf (/home/gregonnet/workbench/projects/qwik-acadamy/kanban-board/node_modules/@builder.io/qwik/server.cjs:575:7)
    at renderMarked (/home/gregonnet/workbench/projects/qwik-acadamy/kanban-board/node_modules/@builder.io/qwik/core.cjs:3497:25)
QWIK Dropping render. State changed during render.
QWIK Dropping render. State changed during render.
QWIK Dropping render. State changed during render.
QWIK Dropping render. State changed during render.
Debugger attached.
```

**Comment by manucorporat** [comment]
Same, let's update to 0.0.36 if you can still reproduce, i will look in detail :)

**Comment by GregOnNet** [comment]
Solved in 0.0.36.
The error messages are completely gone.

Thank you for your support.
